.github/workflows/release-connect-init.yml

```diff
@@ -0,0 +1,92 @@
+name: "[Release] connect NPM and v9"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: "The commit SHA to checkout"
+        required: true
+        type: string
+      deploymentType:
+        description: "Select the deployment type. (example: canary, stable)"
+        required: true
+        type: choice
+        options:
+          - canary
+          - stable
+
+jobs:
+  # Version should have been bumped by now thanks to ./ci/scripts/connect-release-init-npm.js
+  extract-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+          # Checkout the specified commit
+          ref: ${{ github.event.inputs.commit_sha }}
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Extract connect version
+        id: set-version
+        run: echo "version=$(node ./ci/scripts/get-connect-version.js)" >> $GITHUB_OUTPUT
+
+  create-push-release-branch:
+    needs: [extract-version]
+    name: "Create release branch for version ${{ needs.extract-version.outputs.version }}"
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.push-branch.outputs.branch_name }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # Ensure the full commit history is available is required to get specific `ref`.
+          fetch-depth: 0
+          # Checkout the specified commit
+          ref: ${{ github.event.inputs.commit_sha }}
+
+      - name: Create and push new branch
+        env:
+          BRANCH_NAME: "release/connect/${{ needs.extract-version.outputs.version }}"
+        run: |
+          echo ${{ env.BRANCH_NAME }}
+          git checkout -b ${{ env.BRANCH_NAME }}
+          git push origin ${{ env.BRANCH_NAME }}
+          echo "branch_name=${{ env.BRANCH_NAME }}" >> $GITHUB_OUTPUT
+
+  trigger-v9-staging-release:
```

Comment:
Im not sure if this will do what we want, if we trigger the workflows like this it will still be possible to approve the run by one person. I trigger release and i can approve it myself because the deploy was requested by the `github-actions` "user". This is valied for all following trigger jobs

Reply:
I see, I think you are right.
I am researching if there is a way of using the same user that triggered the action as the one that trigger it.

Reply:
I think I have an idea, what if instead of running those workflows using `gh` I will make those workflows templates that will run in the original workflow [release-connect-init.yml](https://github.com/trezor/trezor-suite/pull/12444/files#diff-5d3b3dd645ad50a3cd17a1ab1ff0c12c48f4c39e0a328c4a13adadfa9602e779)  ?

Then it will all be run by the user that is triggering the initial workflow, right?

Reply:
Or we can do the process a bit more manual and trigger those workflows manually  at least for now.

Reply:
Ok, for now I suggest get rid of the automation of triggering all the workflows from one and trigger them manually until we decide how to do it in order to achieve that each of the releases part has to be trigger by one authorize user and approve by other.

Therefore I added this fixup to remove all those trigger... jobs https://github.com/trezor/trezor-suite/pull/12444/commits/3029015c5037b1aef0f990ebcf3b77ee0b1cac18

@vdovhanych if you agree I will squash that fixup and we can try manually then we will see.